concordium-node/Cargo.toml

```diff
@@ -26,6 +26,7 @@ codegen-units = 1
 [dependencies]
 # Internal dependencies
 crypto_common = { path = "../concordium-base/rust-src/crypto_common", features = ["encryption"] }
```

Comment:
Does id export common? If yes, then we can remove this dependency.

Reply:
It does not seem to.